core/src/validator.rs

```diff
@@ -891,6 +892,31 @@ impl Validator {
             ip_echo_server.shutdown_background();
         }
     }
+
+    fn initiate_cost_model(cost_model: &RwLock<CostModel>, cost_table: &[(Pubkey, u64)]) {
+        let mut cost_model_mutable = cost_model.write().unwrap();
+        for (program_id, cost) in cost_table {
+            match cost_model_mutable.upsert_instruction_cost(program_id, cost) {
```

Comment:
Was not familiar with the term `upsert` 😄 , makes sense on clarifying the behavior if key already exists over just using the term "insert"

Reply:
A odd term borrowed from a 3rd database that stuck to my mind, I guess it is short to "update or insert"